### ISSUE TITLE
Struggling with unc0ver “Reload System Deamons” ​

### ISSUE THREAD

**Issue by nuno10costa** [pwn20wndstuff/Undecimus#776]
**Describe the bug**
I'm unable to Jailbreak with the option Reload System Deamons on.

Since several betas ago I was unable to just run the unc0nver tool with the default options selected and achieve a success jailbreak state.
An infinite respring occurs, and the solution I found was to first deselect the Reload Dystem Deamons, then hit jailbreak - this will jailbreak the device whitout any issue - and only after enable the Reload systems Deamons (RSD) and jailbreak again. This workaround worked just fine, with the device being jailbroken with the RSD option on.
However, a few hours ago I updated to unc0ver beta 45, and know the "solution" I mentioned above seems to no longer work.
I always get the infinite respring when selecting the RDS... (I actually went back to the beta 43 but the issue remains).
I found another workaround to make the jailbreak possible and keep the RSD - Jailbreak without RSD, then jailbreak with RSD on but with "Load tweaks" off, and only after jailbreak with all the preset options enabled. This way the jailbreak process completes, even though the springboard crashes afterwards.
​
What does the RSD actually does? Does it have any impact if I just have it off? How do you know for sure if the JB process was completed with the RSD on? I think I did it, but the springboard crashed after the process, so is there a way for us to know if the RSD is actually on?
​
**Device (please complete the following information):**
 - iOS Version: 12.1.2
 - iOS Device: iPhone X
 - unc0ver Version: b45

**Place an "x" between the brackets if true:**
 - [X] this is a bug others will be able to reproduce
 - [X] this issue is present on the latest version of unc0ver

**Comment by clarityzzz** [comment]
RSD faz um ldrestart, enquanto com isso desligado apenas reinicia a SpringBoard. É necessário para que alguns tweaks se injetem corretamente em certos processos. Apenas reiniciar a SpringBoard não permite isso ( e como o novo substrate injeta-se em todo e cada processo, suponho que para um funcionamento correto do substrate, isso também seja necessário ).
Quanto a resoluções, tens o último Rocket Substrate instalado da repo do Ryan Petrich? Tweaks como Eclipse que instalem o AppList?

**Comment by nuno10costa** [comment]
Obrigado pela ajuda.

Sim, tenho o rockerbootstrap atualizado (é esse que referes, certo?).
Sim, tenho o Eclipse instalado, mas antes também tinha... Applist tenho a versão 1.5.14 do Ryan Petrich.

**Comment by drpoutine** [comment]
Duplicate of #717

**Comment by nuno10costa** [comment]
Just figure out the issue.

I suggest who is facing this same problem to go to icleaner pro and disable all tweaks and test the jailbreak with RSD enabled. For me it worked, so it's clear then the issue was with a certain tweak.
Just try enabling them one by one in order to understand which one is causing the problem.

This thread can be closed. Thank you.

**Comment by clarityzzz** [comment]
@nuno10costa isso é o equivalente a fazer jailbreak com o RSD off. Aconselho a fazer apenas isso com o tweak culpado e deixar o RSD on.

**Comment by nuno10costa** [comment]
Sim sim, apenas desactivo o tweak culpado, tudo o resto fica activo. Depois do jailbreak feito (com RDS on), volto a activar o tweak. 
Assim está ok, certo?

**Comment by clarityzzz** [comment]
Sim.

**Comment by nuno10costa** [comment]
Obg. Agora esta-me a acontecer outro problema. Algumas apps bloqueiam no ecrã de entrada, e fecham passado uns segundos. Não as consigo abrir... tentei instalar o CocoaTop e terminar o processo nsurlsessiond (li no Reddit que podia resolver) mas parece q o processo não fecha e não resolve. Apenas resolve voltando a fazer jb. Fazes ideia de como resolver?

**Comment by clarityzzz** [comment]
Manda mensagem no reddit /u/tuaprima ou no discord Clarity#5912
